### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by cdaniluk**

### CODE REVIEW

File: .tflint.hcl [hunk 1]
```diff
@@ -0,0 +1,45 @@
+config {
+  module     = true
+  deep_check = false
```
Comment (sblack4) [inline .tflint.hcl:3]: this attribute is deprecated now

File: elb.tf [hunk 2]
```diff
@@ -35,11 +45,22 @@ resource "aws_security_group_rule" "elb_allow_egress" {
...
   name_prefix                      = substr(var.name, 0, 6)
   enable_cross_zone_load_balancing = true
-  internal                         = false
+  internal                         = false #tfsec:ignore:AWS005
   load_balancer_type               = "application"
   security_groups                  = [aws_security_group.elb.id]
   subnets                          = var.public_subnets
   tags                             = var.tags
+
+  dynamic "access_logs" {
```
Comment (sblack4) [inline elb.tf:54]: are you sure you can do this? The docs say "An Access Logs block", which I interpret as singular. https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb#access_logs

File: main.tf [hunk 3]
```diff
@@ -75,6 +78,19 @@ data "template_file" "this" {
   }
 }
 
+data "template_cloudinit_config" "this" {
```
Comment (sblack4) [inline main.tf:81]: I was confused until I saw that this, by default, renders it as base64 encoded https://registry.terraform.io/providers/hashicorp/template/latest/docs/data-sources/cloudinit_config#base64_encode

**Review by sblack4 — CHANGES_REQUESTED** [review]
I like the `.terraform-version` file

File: main.tf (hunk 3 above)
Comment (sblack4) [reply main.tf:81]: This is not an issue, I just wanted to point this out in case it confuses someone else

File: elb.tf (hunk 2 above)
Comment (cdaniluk) [reply elb.tf:54]: yep, we use this pattern a lot. Pay attention to how I build the local it iterates on.

Comment (sblack4) [reply elb.tf:54]: I don't get it

**Review by sblack4 — APPROVED** [review]

**Pull request merged**